hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/CompactNestedFieldsTest.java

```diff
@@ -44,50 +52,61 @@ public static void beforeClass() {
         initializeWithClient(3, config, null);
     }
 
+    static void setupCompactTypesForNestedQuery(HazelcastInstance instance) {
+        createCompactType(instance, "Office", "id BIGINT", "name VARCHAR");
+        createCompactType(instance, "Organization", "id BIGINT", "name VARCHAR", "office Office");
+
+        createCompactMapping(instance, "test", "UserCompactType",
+                "id BIGINT", "name VARCHAR", "organization Organization");
+    }
+
+    static void createCompactMapping(HazelcastInstance instance, String name, String valueCompactTypeName,
```

Comment:
as in other comment, IMO worth indicating that this only takes parameters for Value. It is not obvious because Key can use the same configuration.

Reply:
https://github.com/hazelcast/hazelcast/pull/25269#discussion_r1363255168